ansible/roles/jenkins-partition/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Yaml files should begin with `---`, but also end with `...` (see http://docs.ansible.com/ansible/latest/YAMLSyntax.html)

It seems those markers are not necessary, but if we're adding a starting marker, let's add also the ending marker. (also applies to other tasks).

Reply:
I'm not sure if it is a part of this task to change our convention. I can change it here, but I'm not sure if it is necessary and futhermore changes in other files should be done in another PR.

Reply:
I've noticed it only on new files, but you're right, we can change the rest in a separate PR